libs/langchain/langchain/document_loaders/csv_loader.py

```diff
@@ -31,51 +31,61 @@ class CSVLoader(BaseLoader):
     """
 
     def __init__(
-        self,
-        file_path: str,
-        source_column: Optional[str] = None,
-        csv_args: Optional[Dict] = None,
-        encoding: Optional[str] = None,
+            self,
+            file_path: str,
```

Comment:
Could you revert the whitespace to the way it was?

Reply:
Could you revert the whitespace to the way it was?

ok I will